genie-core/src/main/java/com/netflix/genie/core/jpa/services/JpaJobSearchServiceImpl.java

```diff
@@ -362,4 +363,16 @@ public String getJobHost(@NotBlank final String jobId) throws GenieException {
             throw new GenieNotFoundException("No job execution found for id " + jobId);
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Long getActiveJobCountForUser(@NotBlank final String user) throws GenieException {
```

Comment:
So for this issue you should be able to add `@Validated` to the class to force the `@NotBlank` to fire. That way you don't need the redundant error checking within the method itself. It should be annotated on other classes.